scripts/badger/bribe_ecosystems.py

```diff
@@ -104,30 +111,48 @@ def main(
             mantissa,  # uint256 amount
         )
 
-    if badger_bribe_in_balancer > 0:
-        prop = web3.solidityKeccak(["address"], [r.balancer.B_20_BTC_80_BADGER_GAUGE])
-        mantissa = int(badger_bribe_in_balancer * Decimal(1e18))
+    def bribe_balancer(gauge, mantissa):
+        prop = web3.solidityKeccak(["address"], [gauge])
+        mantissa = int(mantissa)
 
         badger.approve(bribe_vault, mantissa)
+        print(gauge, prop.hex(), mantissa)
         balancer_briber.depositBribeERC20(
             prop,  # bytes32 proposal
             badger,  # address token
             mantissa,  # uint256 amount
         )
 
-    if badger_bribe_in_convex > 0:
+    if "," in bribes["balancer"]:
```

Comment:
otherwise when passing only one amount getting `TypeError: argument of type 'decimal.Decimal' is not iterable`
```suggestion
    if isinstance(bribes["balancer"], str):
```

Reply:
Ah, true, also get this error and it indeed gets resolved as suggested